### ISSUE TITLE
Edit needed to readme file...

### ISSUE THREAD

**Issue by mattslay** [fdbozzo/foxbin2prg#59]
I think you should remove this text from the readme file, as I think it is not accurate to be here in the master repo:

```
Why this fork?
This is a fork of Fernando D. Bozzo foxbin2prg.
It's main purpose it's to fix some bugs and add some new functions. It looks like Fernando does not longer maintain FoxBin2Prg, so, with his permission, I introduce this fork to VFPX VFPX Picture.

I do not maintain the spanish documentation - it would fade. So just check Fernados repo for this information .
```

**Comment by lscheffler** [comment]
High Matt
You are right, it was not the idea of it.   
Any suggestions how to express that the Spanish part is no longer mainted, or do you think one should remove the spanishg part at all, because it slowly degrades?

**Comment by mattslay** [comment]
I was mainly objecting to the “This is a fork of...”.

 I believe at one time Doug made a fork and added that text, then his fork got pulled back into main here, but this is not a fork at all, for it is the actual main repo again.

**Comment by lscheffler** [comment]
It's the fork if mine. ;)   I will change it with the bug fix.    
@DougHennig You might close this issue.

**Comment by mattslay** [comment]
*This* repo is not a Fork of anything... This is the "fdbozzo/foxbin2prg" repo, and it is not Forked from anything else.
 There may have been forks that were created and perhaps merged back to here, but, that does not make this repo a Fork of anything else.

Am I missing something?

**Comment by lscheffler** [comment]
Nope, you are right at all. It only looks like today I express myself wrong all the time.   
**I** have introduced this on my fork, and it was merged by mistake. I've changed the file on my fork, but it must be merged to the repo here.   
Check the fork at https://github.com/lscheffler/foxbin2prg#readme, this is the file changed.  Doug will merged the bug fix to #58 from my fork, and this will include the readme file. too.
